src/components/Modal/Modal.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+
+import './Modal.css';
+
+export default class Modal extends Component {
+  state = {
+    isActive: false,
+  };
+
+  handleOpen = () => {
+    this.setState({
+      isActive: true
+    });
+    document.body.className += ' is-modal';
```

Comment:
```suggestion
    document.body.classList.add('is-modal');
```

Reply:
Good Idea, I just tried to handle it in low level for support reason.